src/Microsoft.AspNetCore.SystemWebAdapters/HttpResponse.cs

```diff
@@ -252,6 +253,10 @@ private string ResolvePath(string url)
 
         public void SetCookie(HttpCookie cookie) => Cookies.Set(cookie);
 
+        public void Flush() => _response.CompleteAsync().GetAwaiter().GetResult();
+
+        public Task FlushAsync() => _response.CompleteAsync();
```

Comment:
CompleteAsync is more equivalent to End(). Response.Body.Flush/Async() is a closer match.

Reply:
Good call - updated